mt-kahypar/utils/initial_partitioning_stats.h

```diff
@@ -148,13 +148,54 @@ class InitialPartitioningStats {
   size_t _total_sum_number_of_threads;
 };
 
-inline std::ostream & operator<< (std::ostream& str, const InitialPartitioningStats& stats) {
+inline std::ostream & operator<< (std::ostream& str, const InitialPartitioningStatsT& stats) {
   str << " average_number_of_threads_per_ip_call="
       << stats.average_number_of_threads_per_ip_call();
   for ( const InitialPartitionerSummary& summary : stats._ip_summary ) {
     str << summary;
   }
   return str;
 }
+
+class DoNothingInitialPartitioningStats {
```

Comment:
The clean way to handle this would be to integrate it in the context, or some other (non-const) object that gets passed around, and merge at appropriate places. Same goes for timers

Reply:
postponed